node/config/types.go

```diff
@@ -289,6 +289,9 @@ type LocalIndexDirectoryConfig struct {
 	ServiceRPCTimeout Duration
 	// PieceDoctor runs a continuous background process to check each piece in LID for retrievability
 	EnablePieceDoctor bool
+	// Interval in hours after which LID clean up job should rerun. The cleanup entails removing indices and metadata
+	// for the expired/slashed deals. Disabled if set to 0.
+	LidCleanupInterval int
```

Comment:
I am curious why an `int` type? Would it be clearer to use `Duration` type considering it exists and that's what the ticker uses down the line?

Reply:
I have updated to time.Duration(). I had 2 different approaches in mind and merged the 2 which resulted in ticker with duration and var with int.